src/views/Dashboard/Dashboard.tsx

```diff
@@ -4,13 +4,13 @@ import { AccountSummary } from '@/components/AccountSummary';
 import { TransferForm } from '@/components/TransferForm';
 import { TransactionHistory } from '@/components/TransactionHistory';
 import { useWebsocket } from '@/hooks';
-import { usePikeEnabled } from '@/hooks/useFeatureFlags';
+import { usePikePaymentEnabled } from '@/hooks/useFeatureFlags';
 
 export const Dashboard = () => {
   const lgMatch = useMediaMatch('lg');
   useWebsocket();
 
-  const pikeEnabled = usePikeEnabled();
+  const pikePaymentEnabled = usePikePaymentEnabled();
```

Comment:
It should be used `usePikeContactsEnabled();`

Reply:
Done